### PR TITLE
[kokoro] Add a stub apidiff command for the apidiff CI job.

### PR DESCRIPTION
**Pull request by jverkoey**
This command will generate an API diff for the current pull request. In a follow-up PR it will also post the results of the API diff to the PR as a comment.

This is the first of two changes to the `apidiff` branch that will add support for API diff reporting to our PRs. The second PR will add the github comment reporting tool.

### CODE REVIEW

File: .kokoro [hunk 1]
```diff
@@ -228,6 +228,102 @@ generate_website() {
...
+    usage
+    exit 1
+  fi
+
+  BUILDS_TMP_PATH=$(mktemp -d)
+  SEEN_XCODES_FILE_PATH="$BUILDS_TMP_PATH/seen_xcodes"
+  touch "$SEEN_XCODES_FILE_PATH"
+  xcodes=$(ls /Applications/ | grep "Xcode")
+  for xcode_path in $xcodes; do
+    xcode_version=$(cat /Applications/$xcode_path/Contents/version.plist \
```
Comment (ianegordon) [inline .kokoro:283]: Is this simpler with plistbuddy?
plistbuddy -c "Print CFBundleShortVersionString" /Applications/$xcode_path/Contents/version.plist

**Review by ianegordon — APPROVED** [review]
Can we use the existing diff in scripts/release?
I can see these two versions getting out of sync.

File: .kokoro (hunk 1 above)
Comment (jverkoey) [reply .kokoro:283]: Might be - can you file a bug so we can clean this up everywhere?

**Comment by jverkoey** [comment]
This ci job is using scripts/release already.

**Pull request merged**